dpcpp/matrix/dense_kernels.dp.cpp

```diff
@@ -329,8 +330,28 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_DENSE_CONVERT_TO_COO_KERNEL);
 
 
+template <typename ValueType>
+void mem_size_bccoo(std::shared_ptr<const DpcppExecutor> exec,
+                    const matrix::Dense<ValueType>* source,
+                    const size_type block_size,
+                    const matrix::bccoo::compression compress,
+                    size_type* result) GKO_NOT_IMPLEMENTED;
+
+GKO_INSTANTIATE_FOR_EACH_VALUE_TYPE(GKO_DECLARE_DENSE_MEM_SIZE_BCCOO_KERNEL);
+
+
+template <typename ValueType, typename IndexType>
+void convert_to_bccoo(std::shared_ptr<const DpcppExecutor> exec,
```

Comment:
when you under <dpcpp/cuda/hip> namespace, you can use DefaultExecutor.
```suggestion
void convert_to_bccoo(std::shared_ptr<const DefaultExecutor> exec,
```

Reply:
Done!!